core/domain/draft_upgrade_services.py

```diff
@@ -207,6 +207,22 @@ def _convert_html_in_draft_change_list(
                 })
         return draft_change_list
 
+    @classmethod
+    def _convert_states_v49_dict_to_v50_dict(cls, draft_change_list):
+        """Converts draft change list from state version 49 to 50. State
+        version 50 removes rules from explorations that use one of the following
+        rules: [ContainsSomeOf, OmitsSomeOf, MatchesWithGeneralForm]. It also
+        renames `customOskLetters` cust arg to `allowedVariables`.
+
+        Args:
+            draft_change_list: list(ExplorationChange). The list of
+                ExplorationChange domain objects to upgrade.
+
+        Returns:
+            list(ExplorationChange). The converted draft_change_list.
+        """
+        return draft_change_list
```

Comment:
The docstring mentions some edits but nothing is done here? Maybe edit docstring with why nothing needs to be done here.

Reply:
Updated docstring.